tools/ci/activate.sh

```diff
@@ -1,10 +1,10 @@
-if [ -e virtenv/bin/activate ]; then
-    source virtenv/bin/activate
-elif [ -e virtenv/Scripts/activate ]; then
-    source virtenv/Scripts/activate
+if [ -e venv/bin/activate ]; then
```

Comment:
This change was made so that we are consistent in local dev and remote dev with the virtualenv